content/docs/1.6.4/deploy/important-notes/index.md

```diff
@@ -224,14 +224,14 @@ Longhorn v{{< current-version >}} does not allow the following:
 However, Longhorn still allows the listed actions when cloning or restoring volumes created with earlier Longhorn
 versions.
 
-### Backup Related Custom Resources Might Disappear
+### Backup Related Custom Resources In The Cluster And Backup Data On The Remote Backup Server Might Be Deleted
```

Comment:
We can remove `Backup Related Custom Resources In The Cluste` from the title and only focus on the `Backup Data On The Remote Backup Server` instead. The risk is the deletion of the backup data.

```suggestion
### Backup Related Custom Resources In The Cluster And Backup Data On The Remote Backup Server Might Be Deleted
```

Reply:
Modified. Thanks.